src/components/Theme/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { kebabCase } from '~utils/string'
+import { UICustomizationOptions } from '~types/ui-customisation-options'
+
+export const setUICustomisations = (customUI: UICustomizationOptions): void => {
+  const sdkCustomisations = Object.entries(customUI).map(
+    ([property, value]) => `--osdk-${kebabCase(property)}: ${value};`
+  )
+  if (sdkCustomisations.length > 0) {
+    const style = `
+      <style>
+          :root {
+            ${sdkCustomisations.join('\n')}
+          }
+      </style>`
+    document.head.insertAdjacentHTML('beforeend', style)
```

Comment:
Is it safe to use `insertAdjacentHTML` like this?

Reply:
I think so for this use case and the string is escaped. Additionally none of the CSS/Sass these CSS variables are applied to are for `content`.

Reply:
Looks fine to me but might be worth double checking with security, I have a thread with them currently regarding `innerHtml` usage.

Reply:
If you're really worried you could refactor to manually create the node.

```ts
    const style = document.createElement('style');
    style.textContent = `:root { ... }`;
    document.head.appendChild(style);
```

You could also set it directly to the root element as CSS, which is probably as simple as you can get.

```ts
    document.documentElement.style.cssText = sdkCustomisations.join('\n');
```